api/src/main/java/com/example/api/service/validator/ChapterValidator.java

```diff
@@ -0,0 +1,31 @@
+package com.example.api.service.validator;
+
+import com.example.api.dto.request.map.ChapterForm;
+import com.example.api.error.exception.ExceptionMessage;
+import com.example.api.error.exception.RequestValidationException;
+import com.example.api.model.map.Chapter;
+import com.example.api.repo.map.ChapterRepo;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Objects;
+
+@Component
+@Slf4j
+@RequiredArgsConstructor
+public class ChapterValidator {
```

Comment:
W MapValidator dawalem metody do walidowania chaptera, wiec albo przenies je tu, albo to co jest tu tam.